apps/mobile/src/screens/Login/OnboardingEmailScreen.tsx

```diff
@@ -21,10 +24,26 @@ import { magic } from '../../magic';
 const FALLBACK_ERROR_MESSAGE = `Something unexpected went wrong while logging in. We've been notified and are looking into it`;
 
 export function OnboardingEmailScreen() {
+  const [email, setEmail] = useState('');
+  const query = useLazyLoadQuery<OnboardingEmailScreenQuery>(
+    graphql`
+      query OnboardingEmailScreenQuery($emailAddress: Email!) {
+        isEmailAddressAvailable(emailAddress: $emailAddress)
+      }
+    `,
+    {
+      emailAddress: email,
+    }
+  );
+
   const navigation = useNavigation<LoginStackNavigatorProp>();
+  const route = useRoute<RouteProp<LoginStackNavigatorParamList, 'OnboardingEmail'>>();
+
+  const authMethod = route.params.auth;
```

Comment:
hypernit: could we call the param `authMethod` as well? i like how explicit it is

Reply:
Updated!